kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -62,6 +62,9 @@ def kolibri_set_server_time():
 
 @register.simple_tag(takes_context=True)
 def kolibri_bootstrap_model(context, base_name, api_resource, **kwargs):
+    # Don't bootstrap when the debug panel is active.
+    if getattr(settings, 'DEBUG_PANEL_ACTIVE'):
```

Comment:
Rather than injecting knowledge of the panel into the bootstrap code, perhaps adding a setting to Kolibri called `ENABLE_DATA_BOOTSTRAPPING` or something, `True` by default, but turned off in the custom settings file here?